ddl/ddl_tiflash_test.go

```diff
@@ -91,16 +91,16 @@ func (s *tiflashDDLTestSuite) SetUpSuite(c *C) {
 	s.dom.SetStatsUpdating(true)
 
 	log.Info("Mock stat", zap.Any("infosyncer", s.dom.InfoSyncer()))
-	ddl.PollTiFlashInterval = 1000 * time.Millisecond
-	ddl.PullTiFlashPdTick = 60
+	ddl.PollTiFlashInterval.Store(1000 * time.Millisecond)
+	ddl.PullTiFlashPdTick.Store(60)
```

Comment:
I can move them to the beginning. However, these values will be modified in test cases which I think is in a goroutine other than DDL. I think it may also lead to some data race?